ood-portal-generator/sbin/update_ood_portal

```diff
@@ -27,45 +31,76 @@ while true; do
     case "$1" in
         -h|--help) usage ; exit 0 ; shift ;;
         -r|--rpm) RPM=1 ; shift ;;
+        -f|--force) FORCE=true ; shift ;;
         *) break ;;
     esac
 done
 
+
+if ! grep -q "$APACHE" $SUM 1>/dev/null 2>&1; then
+    echo "Generating Apache config checksum file: '${SUM}'"
+    sha256sum $APACHE > $SUM
+fi
+
 echo "Generating Apache config using YAML config: '${CONFIG}'"
 
-NEW_APACHE="$("${BIN}" -c "${CONFIG}")"
-if ! cmp -s <(echo "${NEW_APACHE}") "${APACHE}"; then
-  if [[ -f "${APACHE}" ]]; then
-    BAK_APACHE="${APACHE}.$(date '+%Y%m%dT%H%M%S')"
-    echo "Backing up previous Apache config to: '${BAK_APACHE}'"
-    mv "${APACHE}" "${BAK_APACHE}"
+NEW_APACHE=$(mktemp)
+"${BIN}" -c "${CONFIG}" > $NEW_APACHE
+
```

Comment:
For clarity maybe add comment here i.e. `# check checksum to see if manual modification of Apache occurred`